x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/jwt/JwtRealmAuthenticateTests.java

```diff
@@ -292,27 +292,32 @@ public void testJwtAuthcRealmAuthcAuthzWithAuthzRealms() throws Exception {
     }
 
     /**
-     * Verify that a JWT realm successfully connects to HTTPS server, and can handle an HTTP 404 Not Found response correctly.
+     * Verify that a JWT realm successfully connects to HTTPS server, and can handle Not Found response correctly.
```

Comment:
not exactly sure the purpose of this test ... it appears to be testing that we threw an exception on start up (which would have prevented startup) and then verified that exception matches the test fixture ?? ...anyway I updated to test that a failure when creating the JWT realm still allows the JWT realm to be created and it can be initialized with or without a failure.

Reply:
There is some housekeeping needs to be done for the JWT realm especially around the tests. I created a backlog item (ES-4440) for it but never had chance to work on it.

Reply:
Thanks, I appreciate the prior and planned efforts to clean up this.  I reverted the [changes](https://github.com/elastic/elasticsearch/pull/98154/commits/ac801d5e7e86bd89ded525f8efdcd50d6461813c) to the start up and will address in a follow up. As such the changes to this test have also been reverted.